docs/integrations.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: Integrations
+nav_order: 4
+---
+
+# Integrations
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+---
+
+# Summary
+
+FIM could be integrated with two platforms (one at the same time). The integration will help users to store in the cloud all the events produced by FIM in a easy way. The stored events could be processed and analyzed lately.
+
+---
+
+# Supported integrations
+
+FIM supports two integrations:
+- ElasticSearch 8.x or OpenSearch 2.x
+- Splunk indexer 9.x
+
+---
+
+# ElasticSearch/OpenSearch integration configuration
+
+The ELS/OS integration is set by the `endpoint` configuration section. This section has different options, but the required ones are the following:
+- ElasticSearch/OpenSearch `address`. This address has to be addressable by the FIM node. The `address` usually includes the port that ELS/OS by default uses the 9200, an example of the `address` field: `https://elastic.example.com:9200`.
+{: .note}
+> Review the `insecure` parameter in a testing environment. In the production environment, you should use `insecure: false` or keep it empty as default `false`.
+- Inside the credentials section, it's required to set the `user` and `password` parameters to access the indexer environment.
+  - The `user` parameter has to contain an ELS/OS username with permission to push an object into an index and create an index.
+  - The `password` parameter has to contain the access password of the previously given user.
+
+{: .note}
+> To see all endpoint parameters take a look at [Configuration file]({% link docs/configuration-file.md %})
+
+Those are all the required steps to start working with ElasticSearch or OpenSearch indexer tools. FIM will create the index of the day called `fim-yyyy-mm-dd`. Any produced event that FIM detects will be in this index.
+
+---
+
+# Splunk integration configuration
+
+The Splunk integration is set by the `endpoint` configuration section. This section has different options. In addition, the Splunk integration will require additional steps to receive FIM events into the Splunk indexer. The steps are the following:
+
+First, it is required to configure the Splunk indexer to accept FIM events.
+
+1. At the Splunk home page, click on the `Settings` tab.
+![Splunk home page](../../assets/images/splunk/home.png)
```

Comment:
Screenshots are broken both in my local copy of the repo and on the online preview @okynos 
![image](https://github.com/Achiefs/documentation.achiefs.github.io/assets/14923294/3f5e7696-7862-4fb5-ac4c-00f49e8de4bc)